userportaldatamodel/notification_log.py

```diff
@@ -0,0 +1,43 @@
+from . import Base
+import datetime
+from sqlalchemy import (
+    Integer,
+    String,
+    Column,
+    Table,
+    Boolean,
+    BigInteger,
+    DateTime,
+    Text,
+)
+from sqlalchemy import UniqueConstraint, Index, CheckConstraint
+from sqlalchemy.orm.collections import attribute_mapped_collection
+from sqlalchemy.ext.associationproxy import association_proxy
+from sqlalchemy.dialects.postgresql import ARRAY, JSONB
+from sqlalchemy.orm import relationship, backref
+from sqlalchemy.schema import ForeignKey
+from sqlalchemy.sql import func
+from sqlalchemy.types import LargeBinary
+from sqlalchemy.orm.collections import MappedCollection, collection
+import json
+
+from marshmallow_sqlalchemy import SQLAlchemyAutoSchema
+from marshmallow_sqlalchemy.fields import Nested
+
+class NotificationLog(Base):
+    __tablename__ = "notification_log"
+
+    id = Column(Integer, primary_key=True, nullable=False, autoincrement=True) 
```

Comment:
Also I am confused on the purpose of this table. Is there a specific reason why there needs to be two that I am not seeing?